cmd/drainer/drainer.toml

```diff
@@ -38,6 +38,15 @@ compressor = ""
 # number of binlog events in a transaction batch
 txn-batch = 20
 
+# syn ddl to downstream db
+ddl-sync = true
+
+# set mark identification to downstream db mark table
+mark-status = false
+
+# in AA environment, the unique identification of back and forth synchronization prevents loop synchronization
+channel-id = 1
```

Comment:
channel is ok, it's not a keywork(chan is) and there's no conflict.

Reply:
Channel may be confusing to maintainers since it's a concept that Go developers used in a daily base. In my opinion it's not a good idea to reuse the name "Channel" when we have other names available.